pyaci/core.py

```diff
@@ -195,14 +204,21 @@ def __init__(self, url, session=None, verify=False, disableWarnings=True,
         self._wsMos = defaultdict(deque)
         self._wsReady = Event()
         self._wsEvents = {}
+        self._autoRefresh = False
+        self._autoRefreshThread = None
+        self._login = {}
+        self._autotest = False
```

Comment:
I'm not thrilled to have test specific handling in the product code. We'll leave this for now, and discuss later how to remove it.